profile/README.md

```diff
@@ -0,0 +1,7 @@
+<div align="center">
+    <a href="https://github.com/webpack/webpack-cli">
```

Comment:
Do we want to use `webpack-cli` here? Maybe better link on webpack itself?

Reply:
Ups 🥲 we can link to the docs site instead.